src/install/linux.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Installing Dart on Linux
+short-title: Linux Install
```

Comment:
I'd prefer just "Linux"

Reply:
Retracted this request, since this short-title is also used for tab titles.

Reply:
We agreed offline to keep the "Install" suffix since `short-title` is also used to generate the browser page title (i.e., the title that is used when one creates a bookmark from the page).